<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QVideoWidget" name="videoWidget" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>611</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="start_Camera">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Camera</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_Camera">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Camera</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
